server/Cargo.toml

```
@@ -28,7 +28,7 @@ kona-registry = { workspace = true }
 alloy-primitives = { workspace = true, features = ["map", "serde"] }
 
 # optimism derivation
-optimism-derivation = { git = "https://github.com/datachainlab/optimism-elc", rev = "v0.1.9", default-features = false }
+optimism-derivation = { git = "https://github.com/datachainlab/optimism-elc", rev = "ea8593ef9e4ba254b7f40b3b532a6b1a6faa0964", default-features = false }
```

After this [PR](https://github.com/datachainlab/optimism-elc/pull/32) is merged and tagged `v0.1.10`:
```suggestion
optimism-derivation = { git = "https://github.com/datachainlab/optimism-elc", rev = "v0.1.10", default-features = false }
```